osf/models/metaschema.py

```diff
@@ -79,3 +97,25 @@ def validate_metadata(self, metadata, reviewer=False, required_fields=False):
         except jsonschema.SchemaError as e:
             raise ValidationValueError(e.message)
         return
+
+
+class AbstractFormBlock(ObjectIDMixin, BaseModel):
+    class Meta:
+        abstract = True
```

Comment:
Given that it's only a model, I don't think we need to make this abstract to start. It'll be easy enough to split apart if we ever want to or need to.